cvat/apps/engine/media_extractors.py

```diff
@@ -179,25 +194,48 @@ def _make_name():
 
 class ZipReader(ImageListReader):
     def __init__(self, source_path, step=1, start=0, stop=None):
-        self._zip_source = zipfile.ZipFile(source_path[0], mode='r')
+        self._zip_source = zipfile.ZipFile(source_path[0], mode='a')
         self.extract_dir = source_path[1] if len(source_path) > 1 else None
         file_list = [f for f in self._zip_source.namelist() if get_mime(f) == 'image']
         super().__init__(file_list, step, start, stop)
 
     def __del__(self):
         self._zip_source.close()
 
-    def get_preview(self):
+    def get_preview(self, dimension="2d"):
+        if dimension == "3d":
+            fp = open(os.path.join(os.path.dirname(__file__), 'assets/3d_preview.jpeg'), "rb")
```

Comment:
I believe the source of 3d preview is to be changed in the future to a real preview.

Reply:
I agree. I will vote to leave it unimplemented or generate a real preview during data preparation.

Reply:
In SoW we have chapter **1.2.2. 3D-tasks must contain a thumbnail of the lidar data and a “3D” label** so the content of the thumbnail must be a frame from real data of particular PCD or BIN file

Reply:
We have left the changes as such, will change it to real preview in the next milestone.